composer/lib/dependabot/composer/update_checker/version_resolver.rb

```diff
@@ -41,7 +41,7 @@ def initialize(extensions)
         SOURCE_TIMED_OUT_REGEX =
           /The "(?<url>[^"]+packages\.json)".*timed out/
         FAILED_GIT_CLONE_WITH_MIRROR = /Failed to execute git clone --(mirror|checkout)[^']*'(?<url>.*?)'/
-        FAILED_GIT_CLONE = /Failed to clone (?<url>.*?)/
+        FAILED_GIT_CLONE = /Failed to clone (?<url>[^ ]+)/
```

Comment:
I'm confused how this was working. Since this was a non greedy match, I believe it would always match an empty URL. Yet we have some tests for this and they generally pass 🤔.

Reply:
I removed this change from the PR since it's not needed to get tests green and I want to investigate it a bit more.